src/utils/entrypoint.ts

```diff
@@ -0,0 +1,31 @@
+import { resolve, toFileUrl } from "../../deps.ts";
+import { error } from "../error.ts";
+/**
+ * Parses the entrypoint to a URL.
+ * Ensures the file exists when the entrypoint is a local file.
+ */
+export async function parseEntrypoint(entrypoint: string): Promise<URL> {
+  let entrypointSpecifier: URL;
+  try {
+    entrypointSpecifier =
+      (entrypoint.startsWith("https://") || entrypoint.startsWith("http://"))
+        ? new URL(entrypoint)
+        : toFileUrl(resolve(Deno.cwd(), entrypoint));
+  } catch (err) {
+    error(
```

Comment:
Considering this is a util function, I feel like we should throw an error and let the caller handle it. Thoughts?

Reply:
Idk - we only use this function only for one specific purpose, and I think in that usecase erroring inside of this function makes a lot of sense. I would vote to keep it like it is implemented now.